public/app/features/admin/AdminFeatureTogglesAPI.ts

```diff
@@ -26,37 +60,86 @@ const backendSrvBaseQuery =
     }
   };
 
-export const togglesApi = createApi({
-  reducerPath: 'togglesApi',
-  baseQuery: backendSrvBaseQuery({ baseUrl: '/api' }),
-  endpoints: (builder) => ({
-    getManagerState: builder.query<FeatureMgmtState, void>({
-      query: () => ({ url: '/featuremgmt/state' }),
-    }),
-    getFeatureToggles: builder.query<FeatureToggle[], void>({
-      query: () => ({ url: '/featuremgmt' }),
-    }),
-    updateFeatureToggles: builder.mutation<void, FeatureToggle[]>({
-      query: (updatedToggles) => ({
-        url: '/featuremgmt',
-        method: 'POST',
-        body: { featureToggles: updatedToggles },
+const getK8sAPI = () => {
+  return createApi({
```

Comment:
only real difference here is the urls and the conversion of the body in the patch method